os/include/debug.h

```diff
@@ -190,8 +196,13 @@ Once LOGM is approved, each module should have its own index
  * @details @b #include <debug.h>
  * @since TizenRT v1.0
  */
+#ifdef CONFIG_ARCH_CHIP_IMXRT
```

Comment:
I don't want to add board-specific temporary feature in common standard header.
How about redefine it in arch header?
If there is a common header which almost imxrt source codes includes, we can add below
```
#undef llwdbg
#define llwdbg wdbg
```

Reply:
I understand your concern. But, it is not possible to redefine in arch specific header. In that case our change will reflect only in arch specific files. As you know, dbg is not working properly and so, where ever dbg is used in entire TizenRT code, I want it to be replaced with lldbg. 

We will try to find a fix for the dbg issue. And revert this change as soon as possible.

Reply:
ok. Could you add a comment to say this is a temporary fix?

Reply:
Done.